The patch file for configuring ~/.bashrc.
  git diff --no-index -U5 ${HOME}/.bashrc.orig ${HOME}/.bashrc
--- .bashrc
+++ .bashrc
@@ -6,10 +6,19 @@
 case $- in
     *i*) ;;
       *) return;;
 esac
 
+# set C locale if runnng on local console
+#   ref: https://ja.stackoverflow.com/questions/27221/
+#   ref: https://unix.stackexchange.com/questions/719909/how-to-detect-if-im-connected-on-the-console
+case "$(tty)" in
+  /dev/tty[0-9])
+    export LANG=C.UTF-8
+    ;;
+esac
+
 # don't put duplicate lines or lines starting with space in the history.
 # See bash(1) for more options
 HISTCONTROL=ignoreboth
 
 # append to the history file, don't overwrite it
@@ -41,11 +50,11 @@ case "$TERM" in
 esac
 
 # uncomment for a colored prompt, if the terminal has the capability; turned
 # off by default to not distract the user: the focus in a terminal window
 # should be on the output of commands, not on the prompt
-#force_color_prompt=yes
+force_color_prompt=yes
 
 if [ -n "$force_color_prompt" ]; then
     if [ -x /usr/bin/tput ] && tput setaf 1 >&/dev/null; then
        # We have color support; assume it's compliant with Ecma-48
        # (ISO/IEC-6429). (Lack of such support is extremely rare, and such
@@ -59,10 +59,19 @@ fi
 if [ "$color_prompt" = yes ]; then
     PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
 else
     PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
 fi
+
+if [ -f ~/.bash_envname ]; then
+    . ~/.bash_envname
+fi
+
+if [ -f ~/.bash_prompt ]; then
+    . ~/.bash_prompt
+fi
+
 unset color_prompt force_color_prompt
 
 # If this is an xterm set the title to user@host:dir
 case "$TERM" in
 xterm*|rxvt*)

